workflow/snakemake_rules/download_from_fauna.smk

```diff
@@ -47,11 +47,29 @@ def _get_virus_passage_category(wildcards):
     else:
         return ""
 
+def _get_prioritized_seqs_file(wildcards):
+    prioritized_seqs_file = None
+    for build_name, build_params in config["builds"].items():
+        if build_params["lineage"] == wildcards.lineage:
+            prioritized_seqs_file = build_params.get('prioritized_seqs_file')
+            break
+
+    if prioritized_seqs_file is not None:
+        return prioritized_seqs_file
```

Comment:
Is there a reason to not make the default value of `prioritized_seqs_file` an empty list so you could always return that variable without the condition logic here? I'm assuming you didn't want to change the type of the variable, but just checking... 😄

Reply:
Hah, I definitely was not thinking about typing here 😄 Updated in 9a8f09c8d385fa1d6bd00c9f4d24effeb4ec28ca